portfolio/src/main/webapp/script.js

```diff
@@ -71,13 +71,12 @@ function getComments() {
   url.searchParams.append('number-of-comments', userNumComments);
 
   fetch(url).then(response => response.json()).then(comments => {
-    console.log(comments);
-
     const commentSectionContainer = document.getElementById('comment-section-container');
     commentSectionContainer.innerHTML = '';
-    
-    for(const comment of comments) {
-      commentSectionContainer.appendChild(createCommentElement(comment));
+    console.log(comments);
+    for (const comment of comments) {
+      console.log(typeof comment);
+      commentSectionContainer.appendChild(createCommentElement(comment.text, comment.imgUrl));
```

Comment:
```suggestion
// This might not be syntactically correct javascript, but hopefully you get the picture.
    console.log("List of comments:", comments);
    for (const comment of comments) {
      console.log("Comment type:", typeof comment);
      commentSectionContainer.appendChild(createCommentElement(comment.text, comment.imgUrl));
```
Generally it's a good idea to log a line explaining what you're logging. The larger your project grows and the more logging statements you have the more important this is.

Often times at Google we have logs that are gigantic (100s or 1000s of pages) so it's essential to be able to search for a keyword(s) and find what we're looking for.

Reply:
Thank you! That definitely makes sense. In this case, I may just delete the log. I just wanted to check what type the comment were so I knew how to access their parameters. Now, through the code and the functions I use, the fact that comments are objects is self apparent.